Formula/popte2.rb

```diff
@@ -0,0 +1,37 @@
+class Popte2 < Formula
```

Comment:
Shouldn't the formula be called "PoPoolationTE2" to be consistent with the software full name?

Reply:
I would think the basic consistency expectation is that a formula installs a command by the same name. Otherwise, how would one proceed after the installation?

Reply:
I think that the formula name should be named in the same way as the software, no? I mean, in this case it is a wrapper script around a jar file, but if you type "brew info popoolationte2" you won't find anything...and I expected that the formula is included...